### PR TITLE
[Gamepad article] Remove `GamepadList` refs

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Remove `GamepadList` refs.

### CODE REVIEW

**Review by beaufortfrancois — APPROVED** [review]
LGTM

Quick question: Did max number of characters per line rule changed?

**Comment by tomayac** [comment]
> Quick question: Did max number of characters per line rule changed?

I have my prettier config set to 100, but probably didn't run prettier back when this was created.

**Comment by beaufortfrancois** [comment]
@tomayac See https://github.com/GoogleChrome/web.dev/blob/main/.prettierrc.js#L4

**Comment by tomayac** [comment]
I thought we use 100 for Markdown. I don't mind either way, it's just a setting. Maybe @MichaelSolati for a final word.

**Review by jpmedley — CHANGES_REQUESTED** [review]
Thom,

There are a couple of nits to fix. I don't want to hold up this PR, but wee need to look at shoring up MDN's documentation on this so that we can replace your reference with a link.

Joe

File: src/site/content/en/blog/gamepad/index.md [hunk 1]
```diff
@@ -65,35 +58,31 @@ if ('getGamepads' in navigator) {
 
 ### How the browser represents a gamepad
 
-The browser represents gamepads as `Gamepad` objects.
-A `Gamepad` has the following fields:
+The browser represents gamepads as `Gamepad` objects. A `Gamepad` has the following fields:
```
[the remark this answers is not in the document]
Comment (tomayac) [reply src/site/content/en/blog/gamepad/index.md]: Added the missing properties.

File: src/site/content/en/blog/gamepad/index.md [hunk 2]
```diff
@@ -194,39 +179,31 @@ const pollGamepad = () => {
...
-Dual-rumble effects are defined by four parameters:
-
-- `duration`:
-   Sets the duration of the vibration effect in milliseconds.
-- `startDelay`:
-  Sets the duration of the delay until the vibration is started.
-- `strongMagnitude` and `weakMagnitude`:
-  Set the vibration intensity levels for the heavier and lighter eccentric rotating mass
-  motors, normalized to the range `0.0`–`1.0`.
+The `vibrationActuator` property returns a `GamepadHapticActuator` object, which corresponds to a
```
[the remark this answers is not in the document]
Comment (jpmedley) [reply src/site/content/en/blog/gamepad/index.md:205]: Do you mind [opening a ticket](https://github.com/mdn/browser-compat-data/issues) in the browser compat database. Here's [the JSON file](https://github.com/mdn/browser-compat-data/blob/main/api/Gamepad.json) that needs to be updated.

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: src/site/content/en/blog/gamepad/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (tomayac) [reply src/site/content/en/blog/gamepad/index.md:205]: Will do. Can this PR land meanwhile? We can come back to it in the future.

[the remark this answers is not in the document]
Comment (tomayac) [reply src/site/content/en/blog/gamepad/index.md:205]: @jpmedley FYI, chimed in on https://github.com/mdn/browser-compat-data/issues/12180.

**Review by jpmedley — APPROVED** [review]

**Pull request merged**